images/bootstrap/bootstrap.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
 /usr/local/bin/stolonboot -sentinels 1 -rpc 1
+
+if [[ $(kubectl get nodes -l stolon-keeper=yes -o name | wc -l) -eq 3 ]]
```

Comment:
1. Sure. Just have copied old condition from console. Updated.
2. Yes, it shows only node name line by line.